docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  dataact-postgres:
+    container_name: dataact-postgres
+    image: postgres:9.4.7
```

Comment:
I'm going to go back to our discussion and loop @michaeldhess and @dpb-bah for their opinions on this. I still think we should go straight to postgres 9.5 on this one. I realize we're running 9.4 on dev/staging/prod now but an upgrade is in the plans and we're going to have to update and re-test this as soon as that happens anyway. All the devs already code against 9.5 so I see no reason to downgrade at this point.

Reply:
I'm starting to think otherwise. Unless there's a defined story in an upcoming Sprint to upgrade the Broker DBs, we don't know when it'll actually happen. We've been saying "soon" for months now, so it feels safer to downgrade the local envs until it's something that is actually planned.

Reply:
Looping back around to this, it sounds like we're upgrading to postgres 10 this sprint, do we want this to reflect that so we don't have to go back?

Reply:
I will leave this version until the upgrade process begins, you can always change this value locally if you need to test it with different versions.

Reply:
Either this needs to be updated here before this is merged, or updating this file has to be a subtask on whatever ticket upgrades the DB versions. This can't be left at 9.4 for local dev when our production DBs are 10. This will also involve testing to make sure it doesn't break anything with the Docker.

Reply:
the postgres upgrade hasn't begun yet so our production DBs are still 9.4.7, but i'll defer to @willkjackson if this should be postgres 10 or create subtask when the postgres upgrades begins.

Reply:
This is subtask on the operations ticket to upgrade DB versions. We can drop this part of the review.